holodeck/packagemanager.py

```diff
@@ -77,22 +79,30 @@ def world_info(world_name, world_config=None, initial_indent="", next_indent="
             print(sensor_indent, sensor)
 
 
-def install(package_name):
+def install(package_name, holodeck_path="."):
```

Comment:
I updated get_holodeck_path so the holodeck_path arg is not needed, along with the other addition below